src/file.ts

```diff
@@ -55,3 +56,38 @@ export async function getEntryLink(this: SkynetClient, userID: string, path: str
 
   return await this.registry.getEntryLink(userID, dataKey, opts);
 }
+
+/**
+ * Gets the entry data for the entry at the given data path, for the given
+ * public user ID.
+ *
+ * @param this - SkynetClient
+ * @param userID - The public user ID.
+ * @param path - The data path.
+ * @returns - The entry data.
+ */
+export async function getEntryData(
+  this: SkynetClient,
+  userID: string,
```

Comment:
I've not been following skynet-js for a bit now but I am kind of lost here - seeing this function in `file.ts` file doesn't really help either.

What is userID ? Is that some sort of mysky integration ? Do we have an up to date api docs for skynet-js so I can really double back and read up on how does the interface actually look now?

Reply:
Yeah, this is a MySky integration. We added features too quickly and did not document the file API yet, but it's on the agenda.